conf/messages.en

```diff
@@ -3547,6 +3547,13 @@ assetDisposal.reportedOtherAssetsDisposalList.inset.allOtherAssetsDisposed.parag
 assetDisposal.reportedOtherAssetsDisposalList.pagination.label = Showing {0} - {1} of {2} asset disposals
 assetDisposal.reportedOtherAssetsDisposalList.radios.error.required = Select yes if you need to report another asset disposal
 
+assetDisposal.reportedOtherAssetsDisposalList.view.title = View other asset disposals
+assetDisposal.reportedOtherAssetsDisposalList.view.title.plural = View other asset disposals
+assetDisposal.reportedOtherAssetsDisposalList.view.heading = {0} other asset disposal
+assetDisposal.reportedOtherAssetsDisposalList.view.heading.plural = {0} other asset disposals
+assetDisposal.reportedOtherAssetsDisposalList.row.view.hidden = View details of the disposal of {0} by {1}
```

Comment:
It looks like the content doc doesn't include the `by {1}` part, maybe it's worth checking if they want that in the hidden text or not?